guides/common/modules/con_switching-between-hosts.adoc

```diff
@@ -0,0 +1,7 @@
+[id="con_switching-between-hosts_{context}"]
+= Switching between Hosts
+
+When you are on a particular host in the {ProjectWebUI}, you can navigate between hosts without leaving the page by using the host switcher.
+Click the *⇄* next to the hostname.
```

Comment:
```suggestion
Click the *⇄* icon next to the hostname.
```

Reply:
Either is fine but I should go for original one considering minimalism and standard style guide.

Reply:
In that case, I would do this:
```suggestion
Click *⇄* next to the hostname.
```

Reply:
:+1: